### PR TITLE
Add gop_size option. Release v0.23.0.

### PR DESCRIPTION
**Pull request by shuntrho**

### CODE REVIEW

**Review by Rados13 — APPROVED** [review]

File: lib/membrane_h264_ffmpeg/encoder.ex [hunk 1]
```diff
@@ -136,6 +141,7 @@ defmodule Membrane.H264.FFmpeg.Encoder do
              state.preset,
              state.profile,
              state.max_b_frames,
+             state.gop_size || -1,
```
Comment (bblaszkow06) [inline lib/membrane_h264_ffmpeg/encoder.ex:144]: This is a bit inconsistent with the way `max_b_frames` is handled. Can you please unify that (I'd prefer your, simpler way)

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Documentation is available at [HexDocs](https://hexdocs.pm/membrane_h264_ffmpeg_
 Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
 ```elixir
-  {:membrane_h264_ffmpeg_plugin, "~> 0.22.1"}
+  {:membrane_h264_ffmpeg_plugin, "~> 0.23.1"}
```
Comment (bblaszkow06) [inline README.md:19]: If it's supposed to be 0.23.0, then
```suggestion
  {:membrane_h264_ffmpeg_plugin, "~> 0.23.0"}
```

File: mix.exs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.H264.FFmpeg.Plugin.MixProject do
   use Mix.Project
 
-  @version "0.22.1"
+  @version "0.23.1"
```
Comment (bblaszkow06) [inline mix.exs:4]:
```suggestion
  @version "0.23.0"
```

**Review by bblaszkow06 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (shuntrho) [reply README.md:19]: I rebased on master, there is 0.23.0 version already there.

**Pull request merged**